### PR TITLE
Update and rename HomeController.cs to CustomerController.cs

### LINKED ISSUE #16: Rename HomeController to CustomerController
In the webapp project

### PR DESCRIPTION
**Pull request by brunocasarotti**
It resolves #16

### CODE REVIEW

**Comment by twarsop** [comment]
Although the changes to the Controller are correct, there are changes that need to be made else where in the code that reference the old HomeController.

For example, the "_Customer.cshtml" partial view contains the line:
`<input type="button" class="button" value="Cancel" onclick="location.href='@Url.Action("Index", "Home")'" />`
Where "Home" will need to be changed for "Customer".

There are a couple of other instances where "Home" will need to be changed to "Customer" too in a similar manner.

tl;dr - views need the controller name updating too

**Comment by brunocasarotti** [comment]
Oh sorry, I'll fix it

**Comment by twarsop** [comment]
Thank you for contributing, but this issue has now been fixed
